tests/test_order_Cubatic.py

```diff
@@ -96,6 +96,31 @@ def test_disordered(self):
             op_max, 0.2,
             err_msg="per particle order parameter value is too high")
 
+    def test_valid_inputs(self):
+        with self.assertRaises(ValueError):
+            # t_initial must be greater than t_final
+            freud.order.Cubatic(
+                t_initial=0.001,
+                t_final=5.0,
+                scale=0.95,
+                n_replicates=10)
+
+        with self.assertRaises(ValueError):
+            # t_final must be greater than 1e-6
+            freud.order.Cubatic(
+                t_initial=5.0,
+                t_final=1e-7,
+                scale=0.95,
+                n_replicates=10)
+
+        with self.assertRaises(ValueError):
+            # scale must be less than 1
+            freud.order.Cubatic(
+                t_initial=5.0,
+                t_final=0.001,
+                scale=1.1,
+                n_replicates=10)
+
```

Comment:
Since we are testing all error conditions here we might as well have one for scale less than zero.

Reply:
Great idea! Fixed.